### PR TITLE
Implement autogeneration for third-party licenses topics

### PR DESCRIPTION
**Pull request by dshevtsov**
## This PR is a:

- Content fix or rewrite

## Summary

When this pull request is merged, it will implement autogenerated tables into topic about third-party licenses in the 2.3 documentation. Data sources for the tables are the corresponding `composer.lock` files.

### CODE REVIEW

File: guides/v2.3/release-notes/thirdparty-mccloud.md [hunk 1]
```diff
@@ -1,101 +1,15 @@
 ---
 group: release-notes
-subgroup: Third-party licenses
-menu_title: Magento Commerce (Cloud) third-party licenses
 title: Magento Commerce (Cloud) third-party licenses
```
Comment (hguthrie) [inline guides/v2.3/release-notes/thirdparty-mccloud.md]:
```suggestion
title: Magento Commerce Cloud third-party licenses
```

**Comment by keharper** [comment]
This is awesome. We need to consider other repositories, such as those for Community Engineering projects like MSI, GraphQL, and Page Builder. I spot checked the MSI `composer.json` file and noticed that `"magento/magento2-functional-testing-framework": "2.3.5",` was in the `require-dev` section, but it's not listed in the build Dima provided. 

We should also plan on updating these files every patch release, instead of only at major releases.

**Comment by keharper** [comment]
This is awesome. We need to consider other repositories, such as those for Community Engineering projects like MSI, GraphQL, and Page Builder. I spot checked the MSI `composer.json` file and noticed that `"magento/magento2-functional-testing-framework": "2.3.5",` was in the `require-dev` section, but it's not listed in the build Dima provided. 

We should also plan on updating these files every patch release, instead of only at major releases.

**Comment by dshevtsov** [comment]
@keharper this topic is for third-party packages. The list is filtered to avoid `magento/` packages.

**Comment by dshevtsov** [comment]
@keharper this topic is for third-party packages. The list is filtered to avoid `magento/` packages.

**Comment by keharper** [comment]
Fair enough. That item just jumped out at me. Here's an external one:

` "squizlabs/php_codesniffer": "3.3.1"`

**Comment by dshevtsov** [comment]
running tests

**Pull request merged**